web/requirements.in

```diff
@@ -25,7 +25,7 @@ django-simple-history # history preservation for contents
 
 # Testing
 pytest
-pytest-django
+pytest-django>=4.5
```

Comment:
Both sets of dependencies here were too old to support the functionality needed here, so these got bumped.